nav2_msgs/action/ComputePathToPoses.action

```diff
@@ -1,5 +1,5 @@
 #goal definition
-geometry_msgs/PoseStamped pose
+nav_msgs/Path poses
```

Comment:
I.... don't know how I feel right now about using the `nav_msgs/Path` type for this. In _one_ way of thinking about this, you could consider a set of waypoint goals to be a .... path. However, we generally talk about a path as a set of waypoints a planner creates to a goal, so this is really messing with definitions.

Reply:
Yes, I doubted if using `nav_msgs/Path` or `geometry_smsgs/PoseStamped[]`. You are right; it is not very accurate to call path to a sequence of waypoints.